src/log.h

```diff
@@ -33,6 +33,7 @@
         sr_errinfo_new(err_info, rc, "Callback \"%s\" of plugin \"%s\" for module \"%s\" failed.", \
         cb_name, plugin_name, mod_name)
 
+#define SR_LOG_ERR(...) sr_log(SR_LL_ERR, __VA_ARGS__)
```

Comment:
There is a reason why it is missing, you should be using `sr_errinfo_new()` for errors.

Reply:
done